README.md

```diff
@@ -179,6 +179,17 @@ python data_list/get_meta_file.py
 sh scripts/train.sh
 ```
 
+## ONNX export
+Support for ONNX model exports is now available
```

Comment:
I noticed that onnx and onnxsim also need to be installed, so it's best to add a line here:
```
pip install onnx onnxsim
```